fr/integrations/plugin-packs/procedures/cloud-azure-web-appservice.md

```diff
@@ -0,0 +1,393 @@
+---
+id: cloud-azure-web-appservice
+title: Azure App Service
+---
+
+## Vue d'ensemble
+
+Azure App Service est une solution qui permet de facilement créer, déployer et
+mettre à l'échelle des applications web et APIs sur des conteneurs ou sur Windows
+et Linux.
+
+Le Plugin-Pack Centreon *Azure App Service* s'appuie sur les API Azure Monitor 
+afin de récuperer les métriques relatives au service
+App Service. Il est possible d'utiliser les 2 modes proposés par 
+Microsoft: RestAPI ou Azure CLI.
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+* Instances Azure *App Service*
+
+### Règles de découverte
+
+Le Plugin-Pack Centreon *Azure App Service* inclut un *provider* de découverte d'Hôtes nommé **Microsoft Azure App Service**.
+Celui-ci permet de découvrir l'ensemble des instances *App Service* rattachés à une *souscription* Microsoft Azure donnée:
+
+![image](../../../assets/integrations/plugin-packs/procedures/cloud-azure-web-appservice-provider.png)
+
+> La découverte *Azure App Service* n'est compatible qu'avec le mode 'api'. Le mode 'azcli' n'est pas supporté dans le cadre
+> de cette utilisation. 
+
+Vous trouverez plus d'informations sur la découverte d'Hôtes et son fonctionnement sur la documentation du module:
+[Découverte des hôtes](../../../monitoring/discovery/hosts-discovery.html)
+
+### Métriques & statuts collectés 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--App-Usage-->
+
+| Metric Name                          | Description                                                                       |
+|:-------------------------------------|:----------------------------------------------------------------------------------|
+| appservice.connections.count         | The number of bound sockets existing in the sandbox                               |
+| appservice.assemblies.current.count  | The current number of assemblies loaded across all AppDomains in this application |
+| appservice.handle.count              | The total number of handles currently open by the app process                     |
+| appservice.thread.count              | The number of threads currently active in the app process                         |
+| appservice.appdomains.count          | The current number of AppDomains loaded in this application                       |
+| appservice.appdomains.unloaded.count | The total number of AppDomains unloaded since the start of the application        |
```

Comment:
```suggestion
| Metric Name                          | Description                                                                       | Unit  |
|:-------------------------------------|:----------------------------------------------------------------------------------|:------|
| appservice.connections.count         | The number of bound sockets existing in the sandbox                               | Count |
| appservice.assemblies.current.count  | The current number of assemblies loaded across all AppDomains in this application | Count |
| appservice.handle.count              | The total number of handles currently open by the app process                     | Count |
| appservice.thread.count              | The number of threads currently active in the app process                         | Count |
| appservice.appdomains.count          | The current number of AppDomains loaded in this application                       | Count |
| appservice.appdomains.unloaded.count | The total number of AppDomains unloaded since the start of the application        | Count |
```